tests/integration-tests/test-suites/athenaQuery.spec.ts

```diff
@@ -128,42 +147,42 @@ describe('Athena Query SQL generation and execution', () => {
 
       const csvRows = await downloadResultsFileAndParseData(randomTicketId)
 
+      const expectedName = `[{"nameparts":[{"type":"GivenName","value":"MICHELLE"},{"type":"FamilyName","value":"KABIR"}]}]`
```

Comment:
Could these live on the testData const?  We define them in 2 places in this file

Reply:
Fixed